packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/Confirm/template.success.tsx

```diff
@@ -103,7 +103,7 @@ const Success = (props: Props) => {
           {getLockRuleMessaging(showLock, days, BSPaymentTypes.BANK_TRANSFER)}
         </div>
       </FlyoutContent>
-      <FlyoutFooter>
+      <FlyoutWrapper>
```

Comment:
Instead of removing the FlyoutFooter you can pass the `collapsed` param to it and it will give you the result you want. `<FlyoutFooter collapsed>`

Reply:
If you want extra credit, you could also add some js doc style comments to the FlyoutFooter component describing the params so that when you hover over it, VSCode will show all the available options for that component.